### PR TITLE
#4410 walletConnect landing page redesign

### PR DESCRIPTION
**Pull request by AlexeyTuboltsev**

### CODE REVIEW

File: packages/web/intl/locales/en-en.json [hunk 1]
```diff
@@ -1464,9 +1465,13 @@
   "voting.failed-to-load-proposal": "Failed to load proposal details.",
   "voting.unknown-proposal": "Unknown proposal.",
   "voting.unsupported-proposal-state": "Unsupported proposal state.",
+  "wallet-connect.beta-testing-message": "This feature is in beta testing. Please share your feedback with our  <a href={href} target=\"_blank\">Customer Care</a>.",
```
Comment (Moejoe90) [inline packages/web/intl/locales/en-en.json:1468]: I'm not sure if its the best idea to put the full links here as we already have this as a react component

Comment (AlexeyTuboltsev) [reply packages/web/intl/locales/en-en.json:1468]: yeah I know about that component. The problem here is that you have to split the wording for it in multiple `FormattedMessages`, sometimes in a very awkward way, like this: "Please visit our "--"customer care"--" if you have problems".  And don't forget the white spaces around all of it

Even worse, if we ever will have another language, it's very likely that the word order will change. That means you can't split this in a way to be really cross-language, which kinda defeats the whole idea of having translations.

Comment (Moejoe90) [reply packages/web/intl/locales/en-en.json:1468]: but on the other hand you have maybe unsafe html in the translation file.
Its also very hard to keep track of what you have there. 

So you end up forgetting to add rel="noopener noreferrer for example which is the case now. btw

For safety please use `ExternalLink`

Comment (AlexeyTuboltsev) [reply packages/web/intl/locales/en-en.json:1468]: hm. once again, this makes it unusable as a translation.
done

Comment (AlexeyTuboltsev) [reply packages/web/intl/locales/en-en.json:1468]: done

**Comment by andywalton** [comment]
@AlexeyTuboltsev @glupiawymowka @Moejoe90 

I have two small changes for this PR please:

1. Update the alert icon please. (Will send share with you @AlexeyTuboltsev)

2. Update the alert error text to: `An error occurred while communicating to your remote wallet. Please retry.`

Please also make sure the colour of the text is: `#FF5B5B`

Screenshot 2020-06-17 12 06 18

**Pull request merged**